src/registrar/models/domain.py

```diff
@@ -953,19 +1123,26 @@ def _fetch_cache(self, fetch_hosts=False, fetch_contacts=False):
                 "tr_date": getattr(data, "tr_date", ...),
                 "up_date": getattr(data, "up_date", ...),
             }
-
             # remove null properties (to distinguish between "a value of None" and null)
             cleaned = {k: v for k, v in cache.items() if v is not ...}
 
             # statuses can just be a list no need to keep the epp object
             if "statuses" in cleaned.keys():
                 cleaned["statuses"] = [status.state for status in cleaned["statuses"]]
-            # get contact info, if there are any
+
+            # Registrant should be of type PublicContact
+            if "registrant" in cleaned.keys():
+                # Registrant, if it exists, should always exist in EppLib.
```

Comment:
Do you mean "should always exist in the DB"?

Reply:
Removed this comment as it was unclear and did not have a value proposition